extensions/miro/src/list-boards.tsx

```diff
@@ -0,0 +1,70 @@
+import React, { useEffect, useState } from "react";
+import * as miro from "./oauth/miro";
+import { Board } from "@mirohq/miro-api";
+import { Action, ActionPanel, Detail, Icon, List, showToast, Toast, useNavigation } from "@raycast/api";
+import InviteBoard from "./invite-board";
+import ListMembers from "./list-member";
+
+export default function ListBoards() {
+  const [isLoading, setIsLoading] = useState<boolean>(true);
+  const [items, setItems] = useState<Board[]>([]);
```

Comment:
A suggestion of improvement (but not required to merge): you could use `useCachedPromise` to cache the items to get a really fast first load

Reply:
Implemented this pattern 👍